sdk/storage/storage-blob/samples/typescript/withConnString.ts

```diff
@@ -5,9 +5,11 @@
 import { BlobServiceClient } from "../../src"; // Change to "@azure/storage-blob" in your package
 
 async function main() {
-  // Create Blob Service Client from Connection String
+  // Create Blob Service Client from Account connection string or SAS connection string
+  // Account connection string example - `DefaultEndpointsProtocol=https;AccountName=myaccount;AccountKey=accountKey;EndpointSuffix=core.windows.net`
+  // SAS connection string example - `BlobEndpoint=https://myaccount.blob.core.windows.net/;QueueEndpoint=https://myaccount.queue.core.windows.net/;FileEndpoint=https://myaccount.file.core.windows.net/;TableEndpoint=https://myaccount.table.core.windows.net/;SharedAccessSignature=sasString`
   const STORAGE_CONNECTION_STRING = "";
-  // Only available in Node.js runtime
+  // Note - Account connection string can only be used in NODE.JS runtime.
```

Comment:
Node.js is the official guidance (from Node.js) for how to capitalize that specific Node runtime ("node" is appropriate in most circumstances though, and could be used here in place of "Node.js runtime").